ctru-rs/src/error.rs

```diff
@@ -119,8 +126,9 @@ impl fmt::Display for Error {
             Self::Libc(err) => write!(f, "{err}"),
             Self::ServiceAlreadyActive => write!(f, "Service already active"),
             Self::OutputAlreadyRedirected => {
-                write!(f, "output streams are already redirected to 3dslink")
+                write!(f, "Output streams are already redirected to 3dslink")
```

Comment:
I guess this is personal preference but I would normally default to start error messages with lowercase, in case the user wanted to use it as part of a sentence:
```rs
println!("Internal error: {err}"); // "Internal error: output streams already redirected"
```
That said maybe a user that really cares about the string representation would write their own custom format handler anyway so this isn't a big deal IMO

Reply:
It seems like Rust’s `std` uses lowercase error strings. I believe we should probably stand by that standard. Thanks for the suggestion.